lib/features/modeling/index.js

```diff
@@ -11,7 +13,8 @@ export default {
     'modeling'
   ],
   __depends__: [
-    BehaviorModule
+    BehaviorModule,
+    DiOrderingModule
```

Comment:
Is this necessary, or rather, why don't we do this for the other modules? Previously, we just added them to the `Viewer` or `Modeler`.

Reply:
Yeah, I added it to the Modeler. Viewer makes no sense because you would not export a diagram from there or should I still add it to the viewer just as a precaution?